### PR TITLE
Add SpeedTest module

### PR DESCRIPTION
**Pull request by TW-D**

### CODE REVIEW

File: speedtest/projects/speedtest/src/lib/components/speedtest.component.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <p>
+                <u>Network</u>
+            </p>
+
+            <ul>
+                <li>
+                    <b>Interface</b> : {{ network_interface }}
+                </li>
+                <li>
+                    <b>Connected</b> : {{ (network_connected) ? 'OK' : 'NOK' }}
```
Comment (foxtrot) [inline speedtest/projects/speedtest/src/lib/components/speedtest.component.html:23]: NOK?

File: speedtest/projects/speedtest/src/lib/components/speedtest.component.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+
+        <div *ngIf="network_status">
+            <p>
+                <u>Dependencies</u>
+            </p>
+
+            <ul>
+                <li>
+                    <b>python3-pip</b> : {{ (dependency_python3_pip) ? 'OK' : 'NOK' }}
```
Comment (foxtrot) [inline speedtest/projects/speedtest/src/lib/components/speedtest.component.html:35]: NOK?

**Review by foxtrot — CHANGES_REQUESTED** [review]
Please fix typos

File: speedtest/projects/speedtest/src/lib/components/speedtest.component.html (hunk 1 above)
Comment (TW-D) [reply speedtest/projects/speedtest/src/lib/components/speedtest.component.html]: Corrected

File: speedtest/projects/speedtest/src/lib/components/speedtest.component.html (hunk 2 above)
Comment (TW-D) [reply speedtest/projects/speedtest/src/lib/components/speedtest.component.html]: Corrected

**Pull request closed without merging**